test/fixtures_stage_based_messaging.js

```diff
@@ -0,0 +1,867 @@
+
+module.exports = function() {
+    return [
+
+        // 0: get active subscriptions for cb245673-aa41-4302-ac47-00000001001
+        {
+            "request": {
+                "url": 'http://sbm.localhost:8001/api/v1/subscriptions/',
+                "method": 'GET',
+                "params": {
+                    "identity": 'cb245673-aa41-4302-ac47-00000001001',
+                    "active": 'true'
+                }
+            },
+            "response": {
+                "code": 200,
+                "data": {
+                    "count": 0,
+                    "next": null,
+                    "previous": null,
+                    "results": []
+                }
+            }
+        },
+
+        // 1: get active subscriptions for cb245673-aa41-4302-ac47-00000001002
+        {
+            "request": {
+                "url": 'http://sbm.localhost:8001/api/v1/subscriptions/',
+                "method": 'GET',
+                "params": {
+                    "identity": 'cb245673-aa41-4302-ac47-00000001002',
+                    "active": 'true'
+                }
+            },
+            "response": {
+                "code": 200,
+                "data": {
+                    "count": 1,
+                    "next": null,
+                    "previous": null,
+                    "results": [
+                        {
+                            'url': 'http://sbm.localhost:8001/api/v1/subscriptions/51fcca25-2e85-4c44-subscription-1002',
```

Comment:
This may be a bit unnecessary, but it would be nice to remove all the `.localhost:800x` parts of the urls, so it's just `http://sbm/api/...`, `http://is/api/...`, etc